lua/telescope/previewers.lua

```diff
@@ -39,9 +40,9 @@ local bat_maker = function(filename, lnum, start, finish)
 
   if has_less then
     if start then
-      table.insert(command, {"--pager", string.format("less -RS +%s", start)})
+      table.insert(command, {"--pager", string.format("'less -RS +%s'", start)})
     else
-      table.insert(command, {"--pager", "less -RS"})
+      table.insert(command, {"--pager", "'less -RS'"})
```

Comment:
Required because this is no longer 1 argument. So we have to add quotes

Reply:
use [[]] for that one. I used when ever there is `\\` `' '` `" "` involved in the string.

Reply:
Will not work here because we are not escaping anything. These `'` are meant for `bat` so that `bat` knows that `-RS` is meant for `less` and not `bat`.  Also required when running this is `bash` or `zsh`